packages/xo-server/src/xo-mixins/subjects.mjs

```diff
@@ -231,7 +231,16 @@ export default class {
   }
 
   async getAllUsers() {
-    return this._users.get()
+    const users = await this._users.get()
+
+    // TODO: remove when no longer the email property has been
+    // completely eradicated.
```

Comment:
```suggestion
    // TODO: remove when he email property has been
    // completely eradicated.
```
also shoudldn't we delete  user.email from the user object at least ?

Reply:
I don't think. `user.email` is still used by XO5